flink-core/src/main/java/org/apache/flink/util/AbstractCloseableRegistry.java

```diff
@@ -90,14 +90,14 @@ public final void registerClosable(C closeable) throws IOException {
 	 *
 	 * @param closeable instance to remove from the registry.
 	 */
-	public final void unregisterClosable(C closeable) {
+	public final boolean unregisterCloseable(C closeable) {
```

Comment:
Please add a description of the return parameter. Doesn't need to be a `@param` block, a description in the text is enough. Same for the method below.